Sources/AutoLayout/Constrain.swift

```diff
@@ -41,6 +41,31 @@ public final class ConstraintGroup {
     }
 }
 
+public final class ConstraintGroupToggle<T: Hashable> {
+
+    private var constraintGroups: [T: ConstraintGroup] = [:]
+
+    public init(initial: T? = nil, constraintGroups: [T: ConstraintGroup]) {
+
+        self.constraintGroups = constraintGroups
+
+        if let initial = initial {
+            activate(initial)
+        } else {
+            constraintGroups.forEach { $1.isActive = false }
```

Comment:
Maybe we could have a `public func deactivate` that would turn off all constraint groups and be used here as well